packages/cashier/src/modules/withdrawal-fiat/components/withdrawal-fiat-iframe/withdrawal-fiat-iframe.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Loading } from '@deriv/components';
+import { observer } from '@deriv/stores';
+import { useWithdrawalFiatAddress } from '@deriv/hooks';
+import './withdrawal-fiat-iframe.scss';
+import ErrorStore from '../../../../stores/error-store';
+import Error from '../../../../components/error';
+
+const WithdrawalFiatIframe = observer(() => {
+    const { data: iframe_url, error, isSuccess, resetVerificationCode } = useWithdrawalFiatAddress();
+    const [is_loading, setIsLoading] = React.useState(true);
+
+    React.useEffect(() => {
+        return () => resetVerificationCode(); // redirects the user back to email verification page
+    }, []);
+
+    React.useEffect(() => {
+        setIsLoading(true);
+    }, [iframe_url]);
```

Comment:
@aum-deriv 
not fully understand why we need this useEffect, if you already set is_loading as true when you start render the component, and disable loading when iframe is loaded?

Reply:
@heorhi-deriv
So, when we send a request for the iframe, intially the `iframe_url = undefined`.
But once we have the `iframe_url`, we still need to wait for the HTML to be loaded from the doughflow url. If we remove the loader before doughflow loads, the user will see a blank screen for a couple of seconds as the `<iframe />` is still loading the iframe.
So I am using the `onLoad` event of iframe to check when the iframe is completely loaded and rendered and then disable the loader.
Does this make sense ? 🤔